.github/workflows/pr-docker-build.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Janus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: PR Docker Build
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  REGISTRY: quay.io
+  USERNAME: ${{ vars.QUAY_USERNAME }}
```

Comment:
where are you defining this? and also WHY... when you can just refer to `secrets.QUAY_USERNAME` (assuming you have created that secret in this repo, like you did for `secrets.QUAY_TOKEN`)

Reply:
Good point, it was like that before so I kept it the same. I'll update it.